2019-01-02-11-15 [restartedMain] [org.springframework.data.elasticsearch.repository.support.AbstractElasticsearchRepository] [ERROR] - failed to load elasticsearch nodes : org.elasticsearch.client.transport.NoNodeAvailableException: None of the configured nodes are available: [{#transport#-1}{COvFTjbqSdOaxsAlaVzN7Q}{127.0.0.1}{127.0.0.1:9200}]
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - 具体错误信息:【None of the configured nodes are available: [{#transport#-1}{COvFTjbqSdOaxsAlaVzN7Q}{127.0.0.1}{127.0.0.1:9200}]】
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClientNodesService.ensureNodesAreAvailable(TransportClientNodesService.java:349)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClientNodesService.execute(TransportClientNodesService.java:247)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportProxyClient.execute(TransportProxyClient.java:60)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClient.doExecute(TransportClient.java:381)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:407)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:396)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.action.ActionRequestBuilder.execute(ActionRequestBuilder.java:46)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.getSearchResponse(ElasticsearchTemplate.java:946)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.queryForPage(ElasticsearchTemplate.java:369)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.queryForObject(ElasticsearchTemplate.java:295)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.repository.query.ElasticsearchPartQuery.execute(ElasticsearchPartQuery.java:81)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.doInvoke(RepositoryFactorySupport.java:605)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.lambda$invoke$3(RepositoryFactorySupport.java:595)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.invoke(RepositoryFactorySupport.java:595)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.projection.DefaultMethodInvokingMethodInterceptor.invoke(DefaultMethodInvokingMethodInterceptor.java:59)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:93)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.SurroundingTransactionDetectorMethodInterceptor.invoke(SurroundingTransactionDetectorMethodInterceptor.java:61)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-11-17 [http-nio-8762-exec-8] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:212)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - 具体错误信息:【None of the configured nodes are available: [{#transport#-1}{COvFTjbqSdOaxsAlaVzN7Q}{127.0.0.1}{127.0.0.1:9200}]】
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClientNodesService.ensureNodesAreAvailable(TransportClientNodesService.java:349)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClientNodesService.execute(TransportClientNodesService.java:247)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportProxyClient.execute(TransportProxyClient.java:60)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClient.doExecute(TransportClient.java:381)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:407)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:396)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.action.ActionRequestBuilder.execute(ActionRequestBuilder.java:46)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.getSearchResponse(ElasticsearchTemplate.java:946)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.queryForPage(ElasticsearchTemplate.java:369)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.queryForObject(ElasticsearchTemplate.java:295)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.repository.query.ElasticsearchPartQuery.execute(ElasticsearchPartQuery.java:81)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.doInvoke(RepositoryFactorySupport.java:605)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.lambda$invoke$3(RepositoryFactorySupport.java:595)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.invoke(RepositoryFactorySupport.java:595)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.projection.DefaultMethodInvokingMethodInterceptor.invoke(DefaultMethodInvokingMethodInterceptor.java:59)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:93)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.SurroundingTransactionDetectorMethodInterceptor.invoke(SurroundingTransactionDetectorMethodInterceptor.java:61)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-11-17 [http-nio-8762-exec-7] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:212)
2019-01-02-11-19 [restartedMain] [org.springframework.data.elasticsearch.repository.support.AbstractElasticsearchRepository] [ERROR] - failed to load elasticsearch nodes : org.elasticsearch.client.transport.NoNodeAvailableException: None of the configured nodes are available: [{#transport#-1}{LA47pT9TSdalyawLB6nAog}{127.0.0.1}{127.0.0.1:9200}]
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - 具体错误信息:【None of the configured nodes are available: [{#transport#-1}{LA47pT9TSdalyawLB6nAog}{127.0.0.1}{127.0.0.1:9200}]】
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClientNodesService.ensureNodesAreAvailable(TransportClientNodesService.java:349)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClientNodesService.execute(TransportClientNodesService.java:247)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportProxyClient.execute(TransportProxyClient.java:60)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClient.doExecute(TransportClient.java:381)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:407)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:396)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.action.ActionRequestBuilder.execute(ActionRequestBuilder.java:46)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.getSearchResponse(ElasticsearchTemplate.java:946)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.queryForPage(ElasticsearchTemplate.java:369)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.queryForObject(ElasticsearchTemplate.java:295)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.repository.query.ElasticsearchPartQuery.execute(ElasticsearchPartQuery.java:81)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.doInvoke(RepositoryFactorySupport.java:605)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.lambda$invoke$3(RepositoryFactorySupport.java:595)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.invoke(RepositoryFactorySupport.java:595)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.projection.DefaultMethodInvokingMethodInterceptor.invoke(DefaultMethodInvokingMethodInterceptor.java:59)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:93)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.SurroundingTransactionDetectorMethodInterceptor.invoke(SurroundingTransactionDetectorMethodInterceptor.java:61)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-11-20 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:212)
2019-01-02-13-51 [restartedMain] [org.springframework.data.elasticsearch.repository.support.AbstractElasticsearchRepository] [ERROR] - failed to load elasticsearch nodes : org.elasticsearch.client.transport.NoNodeAvailableException: None of the configured nodes are available: [{#transport#-1}{iSjbfLTfTpGeNFRVdpMI7Q}{127.0.0.1}{127.0.0.1:9200}]
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - 具体错误信息:【None of the configured nodes are available: [{#transport#-1}{iSjbfLTfTpGeNFRVdpMI7Q}{127.0.0.1}{127.0.0.1:9200}]】
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClientNodesService.ensureNodesAreAvailable(TransportClientNodesService.java:349)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClientNodesService.execute(TransportClientNodesService.java:247)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportProxyClient.execute(TransportProxyClient.java:60)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClient.doExecute(TransportClient.java:381)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:407)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:396)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.action.ActionRequestBuilder.execute(ActionRequestBuilder.java:46)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.doCount(ElasticsearchTemplate.java:501)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.count(ElasticsearchTemplate.java:479)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.repository.support.AbstractElasticsearchRepository.search(AbstractElasticsearchRepository.java:198)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - java.lang.reflect.Method.invoke(Method.java:498)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryComposition$RepositoryFragments.invoke(RepositoryComposition.java:359)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryComposition.invoke(RepositoryComposition.java:200)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$ImplementationMethodExecutionInterceptor.invoke(RepositoryFactorySupport.java:644)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.doInvoke(RepositoryFactorySupport.java:608)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.lambda$invoke$3(RepositoryFactorySupport.java:595)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.invoke(RepositoryFactorySupport.java:595)
2019-01-02-13-52 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-13-58 [restartedMain] [org.springframework.data.elasticsearch.repository.support.AbstractElasticsearchRepository] [ERROR] - failed to load elasticsearch nodes : org.elasticsearch.client.transport.NoNodeAvailableException: None of the configured nodes are available: [{#transport#-1}{WsC7288JQca2NOtZuK6d6w}{127.0.0.1}{127.0.0.1:9200}]
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - 具体错误信息:【None of the configured nodes are available: [{#transport#-1}{WsC7288JQca2NOtZuK6d6w}{127.0.0.1}{127.0.0.1:9200}]】
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClientNodesService.ensureNodesAreAvailable(TransportClientNodesService.java:349)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClientNodesService.execute(TransportClientNodesService.java:247)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportProxyClient.execute(TransportProxyClient.java:60)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClient.doExecute(TransportClient.java:381)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:407)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:396)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.action.ActionRequestBuilder.execute(ActionRequestBuilder.java:46)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.doCount(ElasticsearchTemplate.java:501)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.count(ElasticsearchTemplate.java:479)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.repository.support.AbstractElasticsearchRepository.search(AbstractElasticsearchRepository.java:198)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - java.lang.reflect.Method.invoke(Method.java:498)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryComposition$RepositoryFragments.invoke(RepositoryComposition.java:359)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryComposition.invoke(RepositoryComposition.java:200)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$ImplementationMethodExecutionInterceptor.invoke(RepositoryFactorySupport.java:644)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.doInvoke(RepositoryFactorySupport.java:608)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.lambda$invoke$3(RepositoryFactorySupport.java:595)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.invoke(RepositoryFactorySupport.java:595)
2019-01-02-13-58 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-14-16 [restartedMain] [org.springframework.data.elasticsearch.repository.support.AbstractElasticsearchRepository] [ERROR] - failed to load elasticsearch nodes : org.elasticsearch.client.transport.NoNodeAvailableException: None of the configured nodes are available: [{#transport#-1}{9kTfbCooSZ6cPeLoNYmbQQ}{127.0.0.1}{127.0.0.1:9200}]
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - 具体错误信息:【None of the configured nodes are available: [{#transport#-1}{9kTfbCooSZ6cPeLoNYmbQQ}{127.0.0.1}{127.0.0.1:9200}]】
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClientNodesService.ensureNodesAreAvailable(TransportClientNodesService.java:349)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClientNodesService.execute(TransportClientNodesService.java:247)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportProxyClient.execute(TransportProxyClient.java:60)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClient.doExecute(TransportClient.java:381)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:407)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:396)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.action.ActionRequestBuilder.execute(ActionRequestBuilder.java:46)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.doCount(ElasticsearchTemplate.java:501)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.count(ElasticsearchTemplate.java:479)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.repository.support.AbstractElasticsearchRepository.search(AbstractElasticsearchRepository.java:198)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - java.lang.reflect.Method.invoke(Method.java:498)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryComposition$RepositoryFragments.invoke(RepositoryComposition.java:359)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryComposition.invoke(RepositoryComposition.java:200)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$ImplementationMethodExecutionInterceptor.invoke(RepositoryFactorySupport.java:644)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.doInvoke(RepositoryFactorySupport.java:608)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.lambda$invoke$3(RepositoryFactorySupport.java:595)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.invoke(RepositoryFactorySupport.java:595)
2019-01-02-14-16 [http-nio-8762-exec-1] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - 具体错误信息:【None of the configured nodes are available: [{#transport#-1}{9kTfbCooSZ6cPeLoNYmbQQ}{127.0.0.1}{127.0.0.1:9200}]】
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClientNodesService.ensureNodesAreAvailable(TransportClientNodesService.java:349)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClientNodesService.execute(TransportClientNodesService.java:247)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportProxyClient.execute(TransportProxyClient.java:60)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.transport.TransportClient.doExecute(TransportClient.java:381)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:407)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:396)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.elasticsearch.action.ActionRequestBuilder.execute(ActionRequestBuilder.java:46)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.doCount(ElasticsearchTemplate.java:501)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.core.ElasticsearchTemplate.count(ElasticsearchTemplate.java:479)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.elasticsearch.repository.support.AbstractElasticsearchRepository.search(AbstractElasticsearchRepository.java:198)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - java.lang.reflect.Method.invoke(Method.java:498)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryComposition$RepositoryFragments.invoke(RepositoryComposition.java:359)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryComposition.invoke(RepositoryComposition.java:200)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$ImplementationMethodExecutionInterceptor.invoke(RepositoryFactorySupport.java:644)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.doInvoke(RepositoryFactorySupport.java:608)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.lambda$invoke$3(RepositoryFactorySupport.java:595)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.data.repository.core.support.RepositoryFactorySupport$QueryExecutorMethodInterceptor.invoke(RepositoryFactorySupport.java:595)
2019-01-02-14-17 [http-nio-8762-exec-3] [com.cloud.provider.exception.GlobalExceptionHandler] [ERROR] - org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
